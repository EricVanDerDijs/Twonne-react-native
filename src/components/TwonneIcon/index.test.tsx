import React from 'react';
import { TwonneIcon } from './index';
import renderer from 'react-test-renderer';

function mockHandler(): void {
  // tslint:disable-next-line
  console.debug('test text')
}

it('Renders Twonne Logo', () => {
  const tree = renderer.create((
    <TwonneIcon
      fontSize={14}
      onPress={mockHandler}
    />
  )).toJSON();
  expect(tree).toMatchSnapshot();
});
