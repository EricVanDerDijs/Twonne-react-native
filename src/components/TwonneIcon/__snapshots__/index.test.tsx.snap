// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Twonne Logo 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": "white",
      "borderRadius": 30,
    }
  }
>
  <View
    style={null}
  >
    <Text
      style={
        Object {
          "color": "#F37721",
          "fontFamily": "ExpletusSans-BoldItalic",
          "fontSize": 14,
        }
      }
    >
      Twonne
    </Text>
  </View>
</View>
`;
